BIG_BOT/src/hardware/ultrasonicController.py

```diff
@@ -34,7 +36,28 @@ def add_sensor(self, pos: USPosition, echoPin: int, trigPin: int) -> None:
         if any(sensor.pos == pos for sensor in self._sensors):
             raise ValueError(f"Sensor with position {pos} already exists in the controller.")
         self._sensors.append(UltrasonicSensor(pos, echoPin, trigPin))
+        self._enabled_sensors[pos] = True
+
+
+    def enable_sensor(self, pos: USPosition) -> None:
+        """
+        Enable the ultrasonic sensor at the specified position.
 
+        Parameters:
+            `pos` (USPosition): The position of the sensor.
+        """
+        if pos in self._enabled_sensors:
+            self._enabled_sensors[pos] = True
+
+    def disable_sensor(self, pos: USPosition) -> None:
+        """
+        Disable the ultrasonic sensor at the specified position.
+
+        Parameters:
+            `pos` (USPosition): The position of the sensor.
+        """
+        if pos in self._enabled_sensors:
+            self._enabled_sensors[pos] = False
```

Comment:
Maybe add a `else` clause in case the position is not found so that it can print or warn us that we tried to enable/disable a sensor that is not present.
```suggestion
    def enable_sensor(self, pos: USPosition) -> None:
        if pos in self._enabled_sensors:
            self._enabled_sensors[pos] = True
        else:
            print(f"Tried to enable sensor at position {pos}, but was not in the list of enabled sensors.")

    def disable_sensor(self, pos: USPosition) -> None:
        if pos in self._enabled_sensors:
            self._enabled_sensors[pos] = False
        else:
            print(f"Tried to disable sensor at position {pos}, but was not in the list of enabled sensors.")
```